lib/async_hooks.js

```diff
@@ -49,6 +49,9 @@ const init_symbol = Symbol('init');
 const before_symbol = Symbol('before');
 const after_symbol = Symbol('after');
 const destroy_symbol = Symbol('destroy');
+const emitBeforeN = emitHookFactory(before_symbol, 'emitBeforeN');
```

Comment:
I guess in that case the the `S` should also be renamed but I can't think of what it stands for right now. Does it stand for `Script`?